src/fullnode.rs

```diff
@@ -363,9 +363,10 @@ impl Fullnode {
                 }
                 None => FullnodeReturnType::LeaderToLeaderRotation, // value doesn't matter here...
             };
-
+            let mut new: Arc<Bank> = Arc::new(Bank::new_from_parent(self.bank.clone()));
+            std::mem::swap(&mut self.bank, &mut new);
```

Comment:
Too cute

Reply:
@garious the problem is when do we get that bank back and assign it to self, which is what the tvu uses.

i don't see a good synchronization point.

Reply:
nm, it works!